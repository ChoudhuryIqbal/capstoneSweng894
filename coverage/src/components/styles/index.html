<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/components/styles</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="..\..\..\prettify.css" />
    <link rel="stylesheet" href="..\..\..\base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(..\..\..\sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="..\..\..\index.html">All files</a> src/components/styles
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>0/0</span>
      </div>
    </div>
    <p class="quiet">
      Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
    </p>
  </div>
  <div class='status-line low'></div>
<div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file empty" data-value="AccountAddress.styles.js"><a href="AccountAddress.styles.js.html">AccountAddress.styles.js</a></td>
	<td data-value="0" class="pic empty"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file empty" data-value="AccountDetails.styles.js"><a href="AccountDetails.styles.js.html">AccountDetails.styles.js</a></td>
	<td data-value="0" class="pic empty"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file empty" data-value="AccountForm.styles.js"><a href="AccountForm.styles.js.html">AccountForm.styles.js</a></td>
	<td data-value="0" class="pic empty"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file empty" data-value="BottomBtn.styles.js"><a href="BottomBtn.styles.js.html">BottomBtn.styles.js</a></td>
	<td data-value="0" class="pic empty"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file empty" data-value="ClearBtnSty.styles.js"><a href="ClearBtnSty.styles.js.html">ClearBtnSty.styles.js</a></td>
	<td data-value="0" class="pic empty"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file empty" data-value="DeleteProfileBtn.js"><a href="DeleteProfileBtn.js.html">DeleteProfileBtn.js</a></td>
	<td data-value="0" class="pic empty"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file empty" data-value="ErrorText.styles.js"><a href="ErrorText.styles.js.html">ErrorText.styles.js</a></td>
	<td data-value="0" class="pic empty"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file empty" data-value="General.styles.js"><a href="General.styles.js.html">General.styles.js</a></td>
	<td data-value="0" class="pic empty"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file empty" data-value="LogInBtn.styles.js"><a href="LogInBtn.styles.js.html">LogInBtn.styles.js</a></td>
	<td data-value="0" class="pic empty"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file empty" data-value="PrefChoice.styles.js"><a href="PrefChoice.styles.js.html">PrefChoice.styles.js</a></td>
	<td data-value="0" class="pic empty"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file empty" data-value="PrefResult.styles.js"><a href="PrefResult.styles.js.html">PrefResult.styles.js</a></td>
	<td data-value="0" class="pic empty"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file empty" data-value="PrefTop.styles.js"><a href="PrefTop.styles.js.html">PrefTop.styles.js</a></td>
	<td data-value="0" class="pic empty"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file empty" data-value="SignUpBtn.styles.js"><a href="SignUpBtn.styles.js.html">SignUpBtn.styles.js</a></td>
	<td data-value="0" class="pic empty"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file empty" data-value="SkipNav.styles.js"><a href="SkipNav.styles.js.html">SkipNav.styles.js</a></td>
	<td data-value="0" class="pic empty"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file empty" data-value="index.js"><a href="index.js.html">index.js</a></td>
	<td data-value="0" class="pic empty"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

</tbody>
</table>
</div><div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Sat Oct 26 2019 20:14:59 GMT-0400 (Eastern Daylight Time)
</div>
</div>
<script src="..\..\..\prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="..\..\..\sorter.js"></script>
<script src="..\..\..\block-navigation.js"></script>
</body>
</html>
