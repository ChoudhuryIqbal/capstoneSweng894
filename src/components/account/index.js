export * from './AccountButtons'
export * from './AccountDetails'
export * from './AccountAddress'
export * from './AccountForm'
export * from './AccountImage'
export * from './AccountImageUpload'
export * from './AccountLogIn'
export * from './AccountOptions'