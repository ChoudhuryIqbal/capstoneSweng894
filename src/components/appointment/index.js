export * from './AppointmentList'
export * from './AppointmentItem'
export * from './AppointmentDetailView'
export * from './AppointmentEditView'
export * from './AppointmentView'
export * from './AppointmentRenderPickerField'
export * from './BusinessName'
export * from './Calendar'
export * from './Time'
export * from './Status'
export * from './Stylist'
export * from './Service'