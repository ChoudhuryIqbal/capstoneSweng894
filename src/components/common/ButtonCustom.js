import React from 'react';
import { Text, TouchableOpacity } from 'react-native';

const ButtonCustom = ({ onPress, children, buttonStyle, textStyle }) => {
return (
    <TouchableOpacity onPress={onPress} style={buttonStyle}>
      <Text style={textStyle}>
        {children}
      </Text>
    </TouchableOpacity>
  );
};

export {ButtonCustom}
