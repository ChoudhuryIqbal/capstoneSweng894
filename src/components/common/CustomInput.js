import React from 'react';
import { Platform, TextInput, View, Text } from 'react-native';

const CustomInput = ({ value, onChangeText, placeholder, 
     inputStyle, containerStyle, textAlign}) => {
  return (
    <View style={containerStyle} >
      <TextInput
        placeholder={placeholder}
        underlineColorAnroid={Platform.Os === 'ios' ? '' : 'transparent'}
        autoCorrect={false}
        style={inputStyle}
        value={value}
        textAlign={textAlign}
        onChangeText={onChangeText}
      />
    </View>
  );
};


export { CustomInput };
