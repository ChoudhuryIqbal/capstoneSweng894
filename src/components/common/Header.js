import React from 'react';
import {Platform,View,Text} from 'react-native';
 

const Header = ({}) => {
return (
<View style={styles.headerRow}>
<View style={{alignItems:'flex-start'}}>
    <Text style={styles.headerText}>{"Enter Your Location"}</Text>
</View>
</View>

);
};
export default {Header}