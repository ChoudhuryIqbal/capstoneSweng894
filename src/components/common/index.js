export * from './Button';
export * from './Card';
export * from './CardSection';
export * from './Header';
export * from './Input';
export * from './InputCustom';
export * from './Spinner';
export * from './InputNumber';
export * from './ImageButton';
export * from './ButtonCustom';
export * from './CheckBox';
export * from './PickerList';
export * from './InputNumberCustom'
export * from './Time'