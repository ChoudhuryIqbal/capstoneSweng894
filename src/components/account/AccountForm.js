import React from 'react'
import {
    View,
    Text,
    ScrollView
} from 'react-native'
import AccountImage from './AccountImage'
import AccountLogIn from './AccountLogIn'
import AccountOptions from './AccountOptions'
import { ButtonCustom } from '../common'

const ForgotPassword = (props) => {
    if(props.fgLogic){
        return ( 
            <ButtonCustom
                onPress={props.onForgotClick}
                buttonStyle={ClearBtnSty.btnBtmStyle}
                textStyle={ClearBtnSty.txtBtnStyle}
            >
                {props.forgotTxt}
            </ButtonCustom>
        )
    } else {
        return <Text />
    }
}

const AccountForm = (props) => {
    return (
        <ScrollView style={styles.scrollView}>
            <View style={styles.imgSty}>
                <AccountImage
                    imageHolder={props.imageHolder}
                    placeholder={props.placeholder}
                    image={props.image}
                />
            </View>
            <AccountLogIn
                emailValue={props.email}
                emailOnChange={props.onEmailChge}
                pdValue={props.password}
                pdOnChange={props.onPasswordChge}
            />   
            <View style={styles.Column}>
                <Text style={ErrorText.errorTextStyle}>
                    {props.error}
                </Text>
                <props.onLogInButton />
                <ForgotPassword
                   fgLogic={props.fgLogic}
                   onForgotClick={props.onForgotClick}
                   forgotTxt={props.forgotTxt}
                />
                <ButtonCustom
                    onPress={props.onSocialAccountClick}
                    buttonStyle={ClearBtnSty.btnBtmStyle}
                    textStyle={ClearBtnSty.txtBtnStyle}
                >
                    {props.socialAccountTxt}
                </ButtonCustom>
            </View>
            <View>
                <AccountOptions
                    onPress={props.onOtherAccountOptionClick}
                    buttonStyle={BottomBtnSty.btnBtmStyle}
                    textStyle={BottomBtnSty.txtBtnStyle}
                    viewStyle={BottomBtnSty.bottomView}
                    children={props.otherAccountTxt}
                />
            </View>
        </ScrollView>
    )
}


const styles = {
    scrollView: {
      marginHorizontal: 20,
    },
    imgSty: {
        justifyContent: 'center',
        alignItems: 'center',
        padding: 50
    },
    Column: {
        display: 'flex',
        flexDirection: 'column',
        flexBasis: 90,
        justifyContent: 'center',
        alignItems: 'center',
        flex: 1,
        margin: 10
      }
}

const ClearBtnSty = {
    btnBtmStyle: {
        textColor: 'black',
        flex: 1
    },
    txtBtnStyle: {
        color: 'gray'
    },
    bottomView:{
        paddingUp: 10,
        alignItems: 'center',
        borderBottomColor: 'color',
        borderBottomWidth: 2,
        marginBottom: 20,
      }
}

const ErrorText = {
    errorTextStyle: {
        fontSize: 20,
        alignSelf: 'center',
        color: 'red'
      }
}

const LogInBtnSty = {
    textStyle: {
      alignSelf: 'center',
      color: '#fff',
      fontSize: 16,
      fontWeight: '600',
      paddingTop: 5,
      paddingBottom: 5,        
      justifyContent: 'center',
      alignItems: 'center',
    },
    buttonStyle: {
      flex: 1,
      backgroundColor:'#4FA6FD' ,
      borderRadius: 5,
      borderWidth: 1,
      borderColor: '#fff',
      marginLeft: 10,
      marginRight: 10,
      width: 250, 
      justifyContent: 'center',
      alignItems: 'center',
    }
  };

const BottomBtnSty ={
    btnBtmStyle: {
        textColor: 'black',
        flex: 1
    },
    txtBtnStyle: {
        color: 'gray'
    },
    bottomView:{
        paddingUp: 10,
        alignItems: 'center',
        borderTopColor: 'color',
        borderTopWidth: 2,
        marginBottom: 20,
        }
}

export default AccountForm