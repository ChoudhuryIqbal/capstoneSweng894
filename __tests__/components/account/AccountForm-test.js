import React from 'react'
import AccountForm from '../../../src/components/account/AccountForm'
import {ButtonCustom} from '../../../src/components/common/'
import {create} from 'react-test-renderer';

describe('Account Form correctly renderer', () => {
    test('Account Form renders without creashing', () => {
        const onOtherAccount = (type) => {
            return (
                <ButtonCustom>
                    {'Other Account'}
                </ButtonCustom>
            )
        }

        const onLogInButton = (type) => {
            return (
                <ButtonCustom>
                    {'Log-In'}
                </ButtonCustom>
            )
        }

        const accountForm = create(
            <AccountForm
                imageHolder={false}
                placeholder={require('../../../src/image/Placeholder150.png')}
                image={require('../../../src/image/Placeholder150.png')}
                email={'Test'}
                onEmailChge={() => {}}
                errorEmail={'Test'}
                password={'Test'}
                onPasswordChge={() => {}}
                errorPassword={'Test'}
                error={'Test'}
                onLogInButton={() => onLogInButton('L')}
                fgLogic={true}
                onForgotClick={() => {}}
                forgotTxt={'Forgot Password?'}
                onOtherAccountOptionClick={() => onOtherAccount("S")}
                otherAccountTxt={'Create an Account'}
            />
        )
        expect (accountForm.toJSON()).toBeTruthy();
    })
})