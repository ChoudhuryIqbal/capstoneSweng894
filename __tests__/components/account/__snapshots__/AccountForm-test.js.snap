// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Account Form correctly renderer Matches to snapshot 1`] = `
<RCTScrollView
  style={
    Object {
      "marginHorizontal": 20,
    }
  }
>
  <View>
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "padding": 50,
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/image/Placeholder150.png",
          }
        }
      />
    </View>
    <View>
      <View>
        <View
          style={
            Object {
              "alignItems": "flex-start",
              "backgroundColor": "#ffffff",
              "display": "flex",
              "flex": 1,
              "flexBasis": 90,
              "flexDirection": "column",
              "justifyContent": "center",
              "margin": 5,
            }
          }
        >
          <Text
            style={
              Object {
                "flex": 1,
                "fontSize": 16,
                "paddingBottom": 10,
              }
            }
          >
            Email: 
          </Text>
          <TextInput
            allowFontScaling={true}
            autoCorrect={false}
            onChangeText={[Function]}
            placeholder="user.email@dummy.com"
            rejectResponderTermination={true}
            style={
              Object {
                "backgroundColor": "#ffffff",
                "color": "#000",
                "elevation": 1,
                "flex": 2,
                "fontSize": 18,
                "lineHeight": 3,
                "paddingBottom": 20,
                "paddingLeft": 5,
                "paddingRight": 5,
                "shadowOffset": Object {
                  "height": 10,
                  "width": 5,
                },
                "shadowOpacity": 1,
                "shadowRadius": 5,
              }
            }
            underlineColorAndroid="transparent"
            underlineColorAnroid="transparent"
            value="Test"
          />
          <Text
            style={
              Object {
                "bottom": 0,
                "color": "red",
                "elevation": 1,
                "fontSize": 10,
                "justifyContent": "center",
                "marginTop": 10,
                "position": "absolute",
                "textAlign": "center",
              }
            }
          >
            Test
          </Text>
        </View>
      </View>
      <View>
        <View
          style={
            Object {
              "alignItems": "flex-start",
              "backgroundColor": "#ffffff",
              "display": "flex",
              "flex": 1,
              "flexBasis": 90,
              "flexDirection": "column",
              "justifyContent": "center",
              "margin": 5,
            }
          }
        >
          <Text
            style={
              Object {
                "flex": 1,
                "fontSize": 16,
                "paddingBottom": 10,
              }
            }
          >
            Password: 
          </Text>
          <TextInput
            allowFontScaling={true}
            autoCorrect={false}
            onChangeText={[Function]}
            placeholder="*******************"
            rejectResponderTermination={true}
            secureTextEntry={true}
            style={
              Object {
                "backgroundColor": "#ffffff",
                "color": "#000",
                "elevation": 1,
                "flex": 2,
                "fontSize": 18,
                "lineHeight": 3,
                "paddingBottom": 20,
                "paddingLeft": 5,
                "paddingRight": 5,
                "shadowOffset": Object {
                  "height": 10,
                  "width": 5,
                },
                "shadowOpacity": 1,
                "shadowRadius": 5,
              }
            }
            underlineColorAndroid="transparent"
            underlineColorAnroid="transparent"
            value="Test"
          />
          <Text
            style={
              Object {
                "bottom": 0,
                "color": "red",
                "elevation": 1,
                "fontSize": 10,
                "justifyContent": "center",
                "marginTop": 10,
                "position": "absolute",
                "textAlign": "center",
              }
            }
          >
            Test
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "flex": 1,
          "flexBasis": 90,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "alignSelf": "center",
            "backgroundColor": "#ffffff",
            "color": "red",
            "fontSize": 20,
            "justifyContent": "center",
          }
        }
      >
        Test
      </Text>
      <View
        accessible={true}
        focusable={false}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Text>
          Log-In
        </Text>
      </View>
      <View
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "flex": 1,
            "opacity": 1,
            "textColor": "black",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "gray",
            }
          }
        >
          Forgot Password?
        </Text>
      </View>
    </View>
    <View>
      <View
        style={
          Object {
            "alignItems": "center",
            "borderTopColor": "color",
            "borderTopWidth": 2,
            "marginBottom": 20,
            "marginTop": 130,
            "paddingUp": 10,
          }
        }
      >
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "flex": 1,
              "opacity": 1,
              "textColor": "black",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "gray",
                "marginTop": 10,
              }
            }
          >
            Create an Account
          </Text>
        </View>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
